lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -0,0 +1,346 @@
+<?php
+/**
+ * Manage and give access to lizmap configuration.
+ *
+ * @author    3liz
+ * @copyright 2012 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+
+use Lizmap\Project;
+
+/**
+ * @deprecated
+ * @FIXME getXml, getComposer
+ * Verify this methods are not used in external modules so we can delete them without risk, otherwise, we have to implement them
+ * in Project and call it here
+ */
+class lizmapProject
+{
+    /**
```

Comment:
`const CACHE_FORMAT_VERSION = 1;` is missing

Reply:
some methods from QgisProject are also missing:

```
public function getQgisProjectVersion()
public function getRelations()
public function getThemes()
```
Probably others are missing. Check it.